src/js/components/RoutedButton/RoutedButton.js

```diff
@@ -5,7 +5,7 @@ import { Button } from '../Button';
 
 class RoutedButton extends Component {
   static contextTypes = {
-    router: PropTypes.shape({}).isRequired,
```

Comment:
there is also some in the tests for RoutedButton do we need to take out?

Reply:
I didn't remove `isRequired` from places where it would not cause backwards compatibility issues for users